src/js/tech/flash.js

```diff
@@ -26,9 +26,6 @@ class Flash extends Tech {
 
     let { source, parentEl } = options;
 
-    // Create a temporary element to be replaced by swf object
-    let placeHolder = this.el_ = Lib.createEl('div', { id: player.id() + '_temp_flash' });
```

Comment:
I believe the placeholder used to be required for the flash fallback back in the day. I think it can be safely removed now.